docs/src/main/tut/codecs/custom-codecs.md

```diff
@@ -87,7 +87,7 @@ json.as[Map[Foo, Int]]
 
 It's often necessary to work with keys in your JSON objects that aren't idiomatic case class member
 names in Scala. While the standard generic derivation doesn't support this use case, the
-experimental circe-generic-extras module does provide two ways to transform your case class member
+`generic-extras` module does provide two ways to transform your case class member
```

Comment:
We need to keep the 'experimental' verbiage here for now.  The future of `generic-extras` is still unknown and that repo is labeled as 'experimental'.